modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc

```diff
@@ -12,7 +12,7 @@ CREATE ROLE regularUsers;
 [[property-based-access-control]]
 = Property-based access control
 
-Property-based access control grants permissions to users to read node properties based on property/value conditions.
+Property-based access control grants permissions to users to read element properties based on property/value conditions.
```

Comment:
This sentence is kind of wrong even from the start. Or it does not give the full picture, you can use PBAC to do more than grant permission to read properties.

Reply:
Maybe:

Property-based access control grants/denies permission to roles to read or traverse elements based on property/value conditions.